R/lintr-deprecated.R

```diff
@@ -5,11 +5,63 @@
 #'
 #' These functions have been deprecated from lintr.
 #'
+#' - `open_curly_linter()` and `closed_curly_linter()` check that open and closed curly braces
```

Comment:
These descriptions no longer make sense, because the linters described are defunct.